docs/content/en/main/concepts/investigations/edge-multi-cluster copy.md

```diff
@@ -0,0 +1,135 @@
+---
+title: "Edge multi-cluster"
+linkTitle: "Edge multi-cluster"
+weight: 1
+description: >
+  Investigation of edge multi-cluster
+---
+
+Kubernetes offers a state-based management platform to declaratively developing and operating workloads. Kube allows a developer to
+ create workloads that are node-agnostic and can be constrained to operate in places where certain conditions are met. Operators enjoy
+  gaurantee that workloads are rescheduled to work wherever constraints allow them to run. This approach has satisfied the large
+   majority of use cases in the cloud native community.
+
```

Comment:
The relations between the above k8s paragraph and the rest of the doc is not clear to me. I think that in k8s we mainly talk about placing pods on nodes, which also includes constrains, rescheduling, etc..  
In KCP we are talking bout placing workloads on clusters which is different level (e.g., in KCP we don't want to expose Node capabilities or per Pod scheduling)

Reply:
the link is in the first line of the next paragraph - there are more organizations looking for more detailed control over where a workload runs then what is currently available in off-the-shelf Kubernetes.  To me, using KCP and developing KCP-Edge helps us run workloads at multiple locations (large-scale), in a way that is disconnect-tolerant, and perhaps even without intervention (autonomy).  Today, we can't do much of this because the underlying Kube abstractions don't allow for this.  Therefore we need new parameters, abstractions, and controllers to act in a way that compliments edge use-cases (scale, disconnect-tolerant, autonomous).

Reply:
I think I didn't clarify my point very well.. Let me try again. In general this is a proposal for exploration under KCP so the preface should explain how the proposed use case is related to KCP and then what are the specific areas under KCP that need to be explored to address special requirements of the Edge use case. 
In addition, I think the k8s paragraph includes statements that are not fully clear or accurate (k8s is aware of nodes, it's doing pods based scheduling and not "workload" into "places", as well as  the constraint are pod to Nodes and not complicated workloads). 
Maybe we can put instead a paragraph what people are trying to do with KCP and why, and then show why the Edge use case may be a good fit for this architecture + what are the gaps.

Reply:
@clubanderson : while "require more fine-grain control over the distribution of workload" is not an inaccurate statement about edge, I think it omits a critical point.  This involves expanding beyond a single cluster, and explicitly caring about clusters.  This makes a clear distinction from the TMC work, see how its Investigation page stipulates that clusters are relatively unimportant.

Reply:
I made some edits to the first paragraph to more accurately depict how people use Kube and what they expect as outcomes.  I also changed the second paragraph to express there are situations where placement matters and pods can't just be scheduled anywhere to do work.